pkg/config/replica_config.go

```diff
@@ -38,6 +39,7 @@ const (
 	// minSyncPointRetention is the minimum of SyncPointRetention can be set.
 	minSyncPointRetention           = time.Hour * 1
 	minChangeFeedErrorStuckDuration = time.Minute * 30
+	defaultSQLMode                  = mysql.DefaultSQLMode
```

Comment:
Please add comment to show what is the DefaultSQLMode's value.

Reply:
done